### PR TITLE
Use the new API provided by COAM to simplify users

### PR DESCRIPTION
**Pull request by istanishev**

### CODE REVIEW

File: src/components/UserRow.js [hunk 1]
```diff
@@ -87,6 +87,10 @@ class UserRow extends React.Component {
         </tr>;
     }
 
+    isCurrentUser() {
```
Comment (victorcuena) [inline src/components/UserRow.js]: This method is empty

Comment (victorcuena) [reply src/components/UserRow.js]: It should come from the props.

File: src/components/UsersTable.js [hunk 2]
```diff
@@ -160,38 +160,80 @@ class UsersTable extends React.Component {
...
+                }
+            });
+        }
+        return m;
+    }
+
     onAddOrEditUser(user, rolesChanges, isAdmin) {
         let sub = user.principal || user.user_id;
+        if (!sub) {
+            sub = user.profiles ? user.profiles[0].user_id : null;
```
Comment (victorcuena) [inline src/components/UsersTable.js]: user.profiles &&  user.profiles.length would be better

File: src/components/UserRow.js (hunk 1 above)
Comment (istanishev) [reply src/components/UserRow.js]: Thanks ;)

**Pull request merged**